modules/mri_violations/templates/menu_resolved_violations.tpl

```diff
@@ -73,8 +73,8 @@
 
 <div id="tabs" style="background: white">
     <ul class="nav nav-tabs">
-        <li class="statsTab"><a class="statsTabLink" id="onLoad" href="{$baseurl}/mri_violations/">Not Resolved</a></li>
-        <li class="statsTab active"><a class="statsTabLink" href="{$baseurl}/mri_violations/resolved_violations/">Resolved</a></li>
+        <li class="statsTab"><a class="statsTabLink" id="onLoad" href="{$baseurl|default}/mri_violations/">Not Resolved</a></li>
```

Comment:
Are all these `|default` things dependent on php8? Can you send a PR that just upgrades smarty and adds these without all the other changes for other dependencies (ie. phpunit changes..)

Reply:
I'm unsure if it's php or the composer dependency but I can do that.

Reply:
I think it'll make it easier to review and merge if we can do it incrementally.

Reply:
@driusan see https://github.com/aces/Loris/pull/7166 for the separate PR.